src/v1_services/chat_service.py

```
@@ -0,0 +1,279 @@
+    # - task_type: "RETRIEVAL_QUERY"는 검색 질문용 임베딩을 생성
+    #   (문서 저장용은 "RETRIEVAL_DOCUMENT"를 사용)
+    #
+    # ※ v1에서는 한국어 질문을 그대로 임베딩합니다.
+    #   v2에서는 먼저 영어로 번역한 뒤 임베딩하여 검색 정확도를 높였습니다.
+    try:
+        text_input = TextEmbeddingInput(text=query, task_type="RETRIEVAL_QUERY")
+        embeddings = embedding_model.get_embeddings([text_input])
+        query_vector = embeddings[0].values  # 768차원 숫자 배열
+    except Exception as e:
+        print(f"❌ 임베딩 실패: {e}")
+        raise e
```

Ruff (0.15.2)

[warning] Use `raise` without specifying exception name

Remove exception name

(TRY201)